graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/InterfaceGenerator.kt

```diff
@@ -43,7 +50,7 @@ class InterfaceGenerator(config: CodeGenConfig, private val document: Document)
 
         val mergedFieldDefinitions = definition.fieldDefinitions + extensions.flatMap { it.fieldDefinitions }
 
-        mergedFieldDefinitions.forEach {
+        mergedFieldDefinitions.filterSkipped().forEach {
```

Comment:
This is interesting - I suppose this version of graphql does not have fields explicitly declared on union types

Reply:
Correct